src/pages/all-pages.jsx

```diff
@@ -1,3 +1,5 @@
 export { Home } from "./home/home";
```

Comment:
This file should be named as `./index.js`. 
Then while importing this will work too ```import {Home} from `../src/pages` ```

Reply:
Yay! it worked, thank you!